app/src/test/java/com/kickstarter/viewmodels/ProjectViewModelTest.kt

```diff
@@ -435,13 +447,18 @@ class ProjectViewModelTest : KSRobolectricTestCase() {
     }
 
     @Test
-    fun testScrimIsVisible() {
+    fun testScrimIsVisible_whenNativeCheckoutDisabled() {
```

Comment:
Should we test the `scrimIsVisible` when native checkout is enabled as well?

Reply:
yeah that's what happens in the next 2 tests
```   
@Test
fun testScrimIsVisible_whenNotBackedProject() {
        setUpEnvironment(environmentWithNativeCheckoutEnabled())
```

```    
@Test
fun testScrimIsVisible_whenBackedProject() {
        setUpEnvironment(environmentWithNativeCheckoutEnabled())
```
should I rename them?